components/climate/climate_ir.rst

```diff
@@ -62,11 +60,15 @@ submit a feature request (see FAQ).
 +---------------------------------------+---------------------+----------------------+
 | :ref:`ZH/LT-01<zhlt01>`               | ``zhlt01``          | yes                  |
 +---------------------------------------+---------------------+----------------------+
+| :ref:`Arduino-HeatpumpIR<heatpumpir>` | ``heatpumpir``      |                      |
+| library                               |                     |                      |
++---------------------------------------+---------------------+----------------------+
 
 This component requires that you have configured a :doc:`/components/remote_transmitter`.
 
 Due to the unidirectional nature of IR remote controllers, this component cannot determine the
 actual state of the device and will assume the state of the device is the latest state requested.
+The assumed state is being restored at boot (if you set ``esp8266_restore_from_flash: true``).
```

Comment:
`esp8266_restore_from_flash` is deprecated and will be removed in a future version. The correct option is `restore_from_flash` under the `esp8266` component.